dist/src/main/config/broker.standalone.yaml.template

```diff
@@ -187,6 +187,23 @@
       # This setting can also be overridden using the environment variable ZEEBE_BROKER_DATA_SNAPSHOTPERIOD.
       # snapshotPeriod: 15m
 
+      # When the disk usage is above this value all client commands will be rejected.
+      # The value is specified as a percentage of the total disk space.
+      # The value should be in the range (0, 1).
+      # This setting can also be overridden using the environment variable ZEEBE_BROKER_DATA_DISKUSAGECOMMANDWATERMARK
+      # diskUsageCommandWatermark = 0.8
+
+      # When the disk usage is above this value replication will be paused.
```

Comment:
Replication will not be paused right? It is more that the follower will stop accepting appends.

Reply:
This broker will not replicate events from others. So it is like pausing replication, right?

Reply:
The thing is it will still replicate to the other follower when the other one has no full disk. For more me when we say stop replication I think the leader will no longer send.

Reply:
How about re-wording it as "When the disk usage is above this value, this broker will not replicate events from other brokers"?

Reply:
It still sounds for my like this node is the active part - but it is not. As follower it just accepts or not accepts. Idk. Maybe it is just me. @npepinpe any opinion on this?

Reply:
> When the disk usage is above this value, this broker will stop writing replicated events it receives from other brokers

?